dev-docs/bidders/inmar.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: bidder
+title: Inmar
+description: Inmar Bidder Adapter
+
+biddercode: inmar
+userIds: identityLink, pubCommonId
+media_types: banner, video
+safeframes_ok: false
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name        | Scope    | Description                          | Example                                                   | Type      |
+|-------------|----------|--------------------------------------|-----------------------------------------------------------|-----------|
+| `pid`       | required | The placement ID from Inmar.         | `'ADb1f40rmi'`                                            | `string`  |
+| `supplyType`| required | Define if site or app.               | `'site / app'`                                            | `string`  |
```

Comment:
This can't be both 'site' and 'app' right, it has to be one or the other? Can you change the example to just be 'site' - otherwise it looks like you can include a string with both values.